canonical/canonical.com
# feat: /data/valkey and corresponding modal

## Done

- Built page as per [doc](https://docs.google.com/document/d/1zu_5LIjlcTwyJzJfX3jveX2HfMO8XxkXg5A7E5ZjE6g/edit) and [design](https://www.figma.com/design/uJUwbUAGDog4KCMivrMpnl/24.10-Data-Fabric?node-id=1342-6610&node-type=instance&m=dev)
- Added form as per [doc](https://docs.google.com/document/d/1SZ3vEvBow6dBOSzU97uwzgdiMI8_hHbW_Q_RFgJnwJY/edit)
- Added page to sub navigation
- Drive by:
  - Removed stale sub nav templates as we are now using `secondary-navigation.yaml` for these

## QA

- View the site locally in your web browser at: https://canonical-com-1386.demos.haus/data/valkey
- Check against doc and design
- Check modal against doc and see that submission goes through as expected 

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-14286
